gnc/sub8_trajectory_generator/launch/trajectory_generator_params.launch

```diff
@@ -16,15 +16,20 @@
 	<param name="propagation_step_size" value="2.0" type="double"/>
 	<param name="min_control_duration" value="1.0" type="double"/>
 	<param name="max_control_duration" value="100.0" type="double"/>
+	
```

Comment:
Minor nitpick: Explicitly mention units -- Despite it being assumed that we're an all metric shop, we should leave nothing to chance.

[Reason](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter)